app/src/main/java/com/cooper/wheellog/MainActivity.java

```diff
@@ -253,6 +255,12 @@ public void onReceive(Context context, Intent intent) {
                     if (WheelLog.AppConfig.getMibandMode() != MiBandEnum.Alarm) {
                         WheelLog.Notifications.update();
                     }
+                    if (!LoggingService.isInstanceCreated() &&
```

Comment:
При таком условии логи не будут писаться вообще если AutoLoggingWhenIsMoving выключен.
Хейтерам скобок посвящается :D :D :D 
if (!LoggingService.isInstanceCreated() &&                          (!WheelLog.AppConfig.getStartAutoLoggingWhenIsMoving() ||  WheelData.getInstance().getSpeed() > 30) &&                             WheelLog.AppConfig.getAutoLog())
```suggestion
                    if (!LoggingService.isInstanceCreated() &&
```